src/components/inputs/Dates/Calendar/Calendar.tsx

```diff
@@ -35,6 +35,7 @@ export interface Props {
   selectionEnd?: Date;
   onClick?: (date: Date) => void;
   onMouseEnter?: (date: Date | null) => void;
+  locale?: string;
```

Comment:
I think it would be a good idea for us to type this specifically to `[language]-[REGION]` locale subtags and limit to the ones Vimeo officially supports. We probably want to display somethings differently for `en-US` and `en-GB`, for example.

Reply:
Ah! Just saw you added a `SupportedLocales` type. Let's type them all with region for consistency.

Reply:
good point re:regions. a question i have is how to know which regions to include?

i got the list of `SupportedLocales` from the `ViewerContext` `locales` field (where `en` and `es` are regionless) and referenced our dictionary files, which don't include regions for `en` and `es`.